packages/dev/gui/src/2D/controls/inputText.ts

```diff
@@ -812,6 +812,13 @@ export class InputText extends Control {
         this._markAsDirty();
     }
 
+    /**
+     * Allow the user to select all text
+     */
+    public selectAllText() {

```

Comment:
Any reason not to make _selectAllText public ? Wondering if inputTextArea would not benefit from it as well ?

Reply:
good point! I will